applications/fishing-map/src/features/workspace/workspace.selectors.ts

```diff
@@ -105,11 +105,14 @@ export const selectDataviewInstancesResolved = createSelector(
 
     // resolved array filters to url filters
     dataviewInstancesResolved = dataviewInstancesResolved.map((dataviewInstance) => {
-      if (dataviewInstance.config.type === Generators.Type.HeatmapAnimated) {
+      if (
+        dataviewInstance.config &&
+        dataviewInstance.config.type === Generators.Type.HeatmapAnimated
```

Comment:
It is easier to add just the `?` to validate if config exits
```suggestion
        dataviewInstance.config?.type === Generators.Type.HeatmapAnimated
```